### ISSUE TITLE
Fails when run via xfce shortcuts

### ISSUE THREAD

**Issue by phil294** [mviereck/x11docker#177]
Something like `x11docker --xephyr ...image...` works fine, but when I configure it as an application shortcut in xfce like the following,
![](https://i.imgur.com/tnUCVoa.png)
it breaks:
Output contains, among not important lines,
```
x11docker note: --kwin-xwayland: kwin_wayland not found.
x11docker note: --kwin-xwayland: Xwayland not found.
x11docker note: Using X server option --xorg
x11docker note: Could not find Xephyr, Xnest, nxagent,
  xpra, weston+Xwayland or kwin_wayland+Xwayland to run a nested X server.

x11docker note: Current X server :0.0 runs on tty 7.
  Access it with [CTRL][ALT][F7].

bash: [: not a tty: integer expression expected
x11docker note: New Xorg server  will run on tty not a tty.
  Access it with [CTRL][ALT][Fnot a tty].

/usr/lib/Xorg.wrap: Only console users are allowed to run the X server

sed: -e expression #1, char 16: Invalid collation character
Error: Can't open display: :102
```
I guess the process behind the shortcuts was created via `startx` in a tty, thus the confusion. But should that even matter? /usr/bin/Xephyr *is* very much installed. I even printed out the $PATH from a shortcut and it contained /usr/bin (obviously). So that error message is wrong.

Some weird syntax errors are in the above output, not sure whats going on here.

Furthermore, when I at first was confused about the Xorg wrap error message, I put `allowed_users = anybody` into `/etc/X11/Xwrapper.config` which didnt exist beforehand (using Arch btw hurr durr).  This changed this very error to
```
(EE) 
Fatal server error:
(EE) Unrecognized option: vtnot
(EE) 
(EE) 
```

tl;dr: Xephyr should be working here, even though it comes from a tty, but it doesnt.

Thanks once again for this handy script! :-)

**Comment by mviereck** [comment]
Thank you for reporting this!
I've seen this bug in #176, too. The output of command `tty` is parsed wrong.
Can you please try out the latest master version? It includes a fix.

**Comment by phil294** [comment]
Very sweet. Fixed! thanks
